sig-scheduling/CONTRIBUTING.md

```diff
@@ -0,0 +1,77 @@
+# Contributing to SIG Scheduling
+
+Welcome to contributing to SIG Scheduling. We are excited about the prospect of you
+joining our [community](https://github.com/kubernetes/community/tree/master/sig-scheduling)!
+
+SIG Scheduling is responsible for the components that make Pod placement decisions.
+You can read the SIG mission outlined in the [charter](https://git.k8s.io/community/sig-cluster-lifecycle/charter.md).
+
+We have multi areas you can contribute to, PRs, issues, documentations, new ideas,
+helping to answer end-user's questions, attending meetings, all kinds of contributions are welcomed.
+
+## Before you begin
+
+We strongly recommend you to check out the [Kubernetes Contributor Guide](https://github.com/kubernetes/community/tree/master/contributors/guide)
+and [Contributor Cheat Sheet](https://github.com/kubernetes/community/tree/master/contributors/guide/contributor-cheatsheet).
+
+## Getting Started
+
+* If you're a newcomer and have no idea where to start, we have a non-stale pool of issues that are
+available for you:
+  * [first-good-issue](https://github.com/kubernetes/kubernetes/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22+label%3Asig%2Fscheduling+)
+  * [help wanted](https://github.com/kubernetes/kubernetes/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22+label%3Asig%2Fscheduling+)
+
+* If you want to understand the architectural details of scheduler, you can refer to the [design docs](https://github.com/kubernetes/community/tree/master/contributors/devel/sig-scheduling)
```

Comment:
Actually, there are pretty old docs only.
Instead of (or in addition to) them, I'd prefer to have a link for some official doc explaining the scheduler like:
https://kubernetes.io/docs/concepts/scheduling-eviction/scheduling-framework/

Reply:
> Actually, there are pretty old docs only.

We should update them.

I will mention the scheduling-eviction as well. Thanks.